irr/src/os.h

```diff
@@ -10,6 +10,21 @@
 #include "ILogger.h"
 #include "ITimer.h"
 
+// CODE_UNREACHABLE(): Invokes undefined behavior for unreachable code optimization
+#if defined(__cpp_lib_unreachable)
+#define CODE_UNREACHABLE() std::unreachable();
```

Comment:
remove one `;` or you have two when it's expanded

Reply:
Fixed in 43b8261a1